apps/web/modules/team/type-view.tsx

```diff
@@ -65,6 +67,8 @@ function Type({
         )}
         orgBannerUrl={orgBannerUrl}
         teamMemberEmail={teamMemberEmail}
+        crmOwnerRecordType={crmOwnerRecordType}
+        crmAppSlug={crmAppSlug}
```

Comment:
Pass these new props to the booker so they can make their way to `handleNewBooking`